partials/confirm_login.html

```diff
@@ -0,0 +1,51 @@
+<% extends "partials/base/base.html" %>
+
+<% block content %>
+<!--Header Area-->
+<tr style="page-break-before: always">
+  <td valign="top">
+    <p class="primary" style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0 0 24px 0; text-align: center;"><% block verify_content %>{{t "You signed in to Firefox from %(device)s, %(location)s."}} {{t "Please confirm this sign-in to begin syncing with %(device)." }}<% endblock %></p>
```

Comment:
Thanks @vbudhram. I dig it. Now I'm going to be a jerk. I propose we add location in phase 2, not the initial roll-out, at least if we go for the end of the week release. My reasoning is that it'll add some amount of complexity, and if we go for the short dev timeline, we'll want as few moving parts as possible. Keeping two strings, one with location, one without, should provide us with sufficient flexibility w.r.t. l10n.